### ISSUE TITLE
[error] cannot initialize git

### ISSUE THREAD

**Issue by avolgha** [avolgha/astato#1]
**problem**

when creating typescript project, git is not initialized properly and an error is thrown.

```shell
  ✖ Initializing git
    → sh: line 1: cd  && git init: command not found
```

**what could be wrong?**

seeing that `cd` command has no directory that could be a problem, otherwise cannot execute `sh` command?

**Comment by avolgha** [comment]
fixed in [commit #5c5fbfd](https://github.com/avolgha/astato/commit/5c5fbfdf3b25f1606e2a9d488dccb2c73e477844)

**Comment by avolgha** [comment]
> **fix**
> 
> instead of executing a shell command with a cd command in it that also depends on your current operating system, we just run the git repository initialize command with the project  directory path as argument. So much simpler!
